File: C++ Code/Basics/Array/1 D/Smallest Element.cpp
#include<iostream>
using namespace std;
int main(){
    int n,ans[2]={INT_MAX,0};
    cout<<"Enter the size of array : ";
    cin>>n;
    int arr[n];
    cout<<"Enter the value of elements : "<<endl;
    for(int i=0;i<n;i++)
        cin>>arr[i];
    for(int i=0;i<n;i++){
        if(arr[i]<ans[0]){
            ans[0]=arr[i];
            ans[1]=i;
        }
    }
    cout<<"Element of the value "<<ans[0]<<" in index "<<ans[1]<<" is the smallest element.";
    return 0;
}

File: Java Code/Basics/Array/1 D/SmallestElement.java
import java.util.*;

public class SmallestElement{
  public static void main(String[] args){
    Scanner sc=new Scanner(System.in);
    System.out.print("Enter the number of elements : ");
    int n=sc.nextInt();
    int[] arr=new int[n];
    System.out.println("Enter all the elements :");
    for(int i=0;i<n;i++)
      arr[i]=sc.nextInt();
      int idx=0,ans=arr[0];
    for(int i=1;i<n;i++){
      if(arr[i]<ans){
        ans=arr[i];
        idx=i;
      }
    }
    System.out.println("Smallest element in the given array is "+ans+" at index "+idx);
  }
}